src/libraries/System.Runtime/tests/System/Reflection/NullabilityInfoContextTests.cs

```diff
@@ -282,6 +283,43 @@ public void GenericDictionaryPropertyTest(string propertyName, NullabilityState
             Assert.Null(nullability.ElementType);
         }
 
+        [ConditionalFact(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
+        public void VerifyIsSupportedThrows()
```

Comment:
It might be good to add a trimming test - mostly to verify that if the feature is on, it still works even when trimmed.

Reply:
👍 done.